apps/gallery/locales/gallery.en-US.properties

```diff
@@ -133,3 +133,6 @@ delete-button.ariaLabel = Delete
 
 # Edit image preview
 editImagePreview.ariaLabel = Edit image preview
+
+# Shrinking UI
+shrinking-tip=Swipe to share
```

Comment:
Also, if we're eventually going to make this change in all of the media apps, then we'll probably want a shared locale file rather than adding strings to each individual app.